### ISSUE TITLE
Backuppc broken after Ubuntu upgrade 20.04 -> 22.04

### ISSUE THREAD

**Issue by sirg** [backuppc/backuppc#484]
I've had a working backuppc on Ubuntu server for 10+ years now. Backing up localhost and one remote Linux server both using "rsync". No trouble across multiple OS upgrades. Stable as a rock. Thank you!

After upgrade from Ubuntu 20.04 to 22.04 (which upgraded backuppc from 3.3.2 to 4.4.0 with no apparent errors) no backups succeed:
- initially got "xfer start failed: $Conf{RsyncBackupPCPath} is set to , which isn't a valid executable"
- after adding the path to rsync_bpc and setting PoolV3Enabled =1, I am getting "No files dumped for share <first share in list>" on a full backup started manually.

The docs say "V3 migration: nothing specific is needed..."

Any suggestions on how to troubleshoot from here or what else needs fixing?

**Comment by xtaran** [comment]
Have you read the package's `/usr/share/doc/backuppc/UPGRADING` as advised in `/usr/share/doc/backuppc/NEWS.Debian.gz`? You usually also need to update the value of `$Conf{RsyncClientPath}` and add `$Conf{RsyncSshArgs}`. See also #132.

And yes, major version bumps usually imply some changes. For me, v4 is much more reliable than v3, probably due to the new on-disk format.

**Comment by sirg** [comment]
Thanks, [xataran](https://github.com/xtaran) - hadn't thought to look in `/usr/share/doc/backuppc`.
After looking at both the old and new config files in detail, it looked like too many changes would be required to 'upgrade' the old config. Ended up carefully detailing changes from the original installed config and figuring out the corresponding changes to the new config. Pretty easy, since I made very few changes - but I am not 100% certain I caught everything.
For hosts I ended up wiping out and re-creating each host file from scratch and re-applying equivalents of 3.x customizations. The override checkboxes in the web UI were extremely helpful. Also used the `sudo -p` hack for localhost. The separation of `RsyncArgs` and `RsyncArgsExtra` makes things easier.
Backups on localhost now in progress.
Unfortunately, even though I only ended up changing about a half a dozen entries in `config.pl`, after saving it, I see over 600 lines of diffs from the original version: whitespace changes, line breaks, quoting, reformatting, different perl false-y representations for undefined values, etc.
